src/utils/samlUtils.ts

```diff
@@ -1043,7 +1046,7 @@ export const assertionValidation = (
                 )
               )
             ),
-            E.chain(() =>
+            E.chainFirst(() =>
```

Comment:
ditto

Reply:
fixed in https://github.com/pagopa/io-spid-commons/pull/121/commits/38b94c3f8e8abe80e91d40c449de14b28faff164